lib/constants.ts

```diff
@@ -149,6 +149,8 @@ export const supportedLifecycleRules = [
     'expiration',
     'noncurrentVersionExpiration',
     'abortIncompleteMultipartUpload',
+    'transition',
```

Comment:
https://github.com/scality/backbeat/blob/b1de3712f7f01eae58efb7cfbc5bd751da2ab6dc/extensions/lifecycle/tasks/LifecycleTask.js#L575
```suggestion
    'transitions',
```